architecture_decisions/0003-index_merge-holdings-info.md

```diff
@@ -0,0 +1,11 @@
+# Merging holdings information for indexing
```

Comment:
I think this file should be called "index-merged-holdings-info.md". The difference between "merge" and "merged" here is meaningful and this change will clarify. Also consistency in using dashes instead of underscores.